csm/conf/setup.py

```diff
@@ -716,6 +755,7 @@ def post_install(self, args):
         """
         try:
             self._verify_args(args)
+            # NKP Get csm user credential from salt and decrypt password and set to variables 
```

Comment:
@navalkp Please check whether this comment is required. If not remove it.

Reply:
will remove